### ISSUE TITLE
reverse chronological order

### ISSUE THREAD

**Issue by sahilseth** [gokceneraslan/awesome-deepbio#34]
This is a awesome list! I follow and read through it off and on. Since the number of articles have really grown and would continue to grow. 

Would you like to see this in reverse chronological order, such that latest `awesome` articles are on top? And we don't have to scroll down for additions - may be being too lazy here :)

**Comment by skyfallen** [comment]
I think it is a nice idea, plus I wanted also to propose to break down the list into subsets, e.g. medical imaging, DNA & RNA binding, gene expression etc. They all could be in reverse chronological order as @sahilseth  mentioned, so that it would be easier to check a specific field for updates. Here are few examples: https://github.com/terryum/awesome-deep-learning-papers or https://github.com/songrotek/Deep-Learning-Papers-Reading-Roadmap.

**Comment by iron-lion** [comment]
I think @skyfallen 's idea is good. 
If it is difficult to break down all articles into specific subset. How about add '#tag' things on the end of each item?? like 'CNN, RNA binding'.

**Comment by zafarali** [comment]
@iron-lion I think Subsets work but they don't fix the overall problem of sorting. I think we might be able to break it down into the following categories:

- Bioinformatics (Sequencing tech, protein folding prediction, GWAS)
- Medical Imaging  
- Bio-Signal Analysis (I'm not sure if we have this already)
- Neuroscience
- Oncology 

We can then add tags to them (CNN, RNN, LSTM etc) based on what model was used to help people search. This could be potentially high reward! What do you think?

**Comment by alxndrkalinin** [comment]
I like reverse chronological order idea, common list of papers helps noticing newest papers even if they're not in my category of interest. And I think usage of tags would be great solution for breaking down papers by category, DL models, etc., because they allow one-to-many relationship. Since many papers can fall in multiple categories, category itself can be a type of tag, e.g. paper on analysis of breast cancer images can have `cat:oncology` and `cat:imaging` category tags + `model:cnn` and `model:lstm` model tags all the same time. This will allow effective filtering when looking for specific things.

**Comment by gokceneraslan** [comment]
I agree to @alxndrkalinin tags and rev. order make sense. PRs are welcome :)
